packages/yoastseo/src/markers/index.js

```diff
@@ -1,5 +1,7 @@
 import removeMarks from "./removeMarks";
+import addMark from "./addMark";
 
 export {
+	addMark,
```

Comment:
I see why we expose `addMark` again. It makes a lot of sense.

Do you think that we should remove it again in the future once we stop supporting the older version?

In that case, let's make a (technical debt) issue for it.

Reply:
I would say that there’s a high chance we will need addMark again when we move/add other assessments to Premium. So maybe it’s better to keep it, even if it’s not always used, than to expose and unexpose and expose again? 😄